apps/mobile/shared/lib/bridge.ts

```
@@ -18,70 +17,138 @@ export const appBridge = bridge({
-    if (type === 'success' || type === 'warning' || type === 'error') {
-      await Haptics.notificationAsync(hapticMap[type] as Haptics.NotificationFeedbackType);
-    } else {
-      await Haptics.impactAsync(hapticMap[type] as Haptics.ImpactFeedbackStyle);
+        if (!accessToken) {
+          throw new Error('애플 로그인 응답에 accessToken이 없습니다.');
+        }
+
+        await authStorage.setTokens(accessToken, refreshToken || '');
+
+        return {
+          success: true,
```

Biome (2.3.14)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)